src/main/java/com/bgsoftware/superiorskyblock/service/placeholders/PlaceholdersServiceImpl.java

```diff
@@ -183,7 +183,7 @@ public class PlaceholdersServiceImpl implements PlaceholdersService {
                     .put("bank_limit", (island, superiorPlayer) -> Formatters.NUMBER_FORMATTER.format(island.getBankLimit()))
                     .put("bank_limit_format", (island, superiorPlayer) ->
                             Formatters.FANCY_NUMBER_FORMATTER.format(island.getBankLimit(), superiorPlayer.getUserLocale()))
-                    .put("island_coop_online", (island, superiorPlayer) -> String.valueOf(island.getCoopPlayers().stream().filter(SuperiorPlayer::isOnline).count()))
+                    .put("online_coop", (island, superiorPlayer) -> String.valueOf(island.getCoopPlayers().stream().filter(SuperiorPlayer::isOnline).count()))
```

Comment:
This seems to be redundant, as coop players must be online.
As soon as they leave, they'll be uncoop'ed.